packages/itwinui-css/src/checkbox/checkbox.scss

```diff
@@ -65,10 +64,9 @@
   --_iui-checkbox-background-color: var(--iui-color-background-1);
   --_iui-checkbox-mask-image: initial;
 
+  @include iui-icon-style('m');
```

Comment:
Because the `iui-icon-style` mixin includes `flex-shrink: 0`, we can remove it from this class

Reply:
Thanks, fixed